### ISSUE TITLE
[FGAP] Implement `AdminPermissionEvaluator` for FGAP V2

### ISSUE THREAD

**Issue by vramik** [keycloak/keycloak#35018]
Once the design is finished within [#34921](https://github.com/keycloak/keycloak/issues/34921) it is needed to provide an implementation for the rest of the `AdminPermissionEvaluator`

**Comment by vramik** [comment]
Closing in favor of 
https://github.com/keycloak/keycloak/issues/35568
https://github.com/keycloak/keycloak/issues/35562
https://github.com/keycloak/keycloak/issues/35564
https://github.com/keycloak/keycloak/issues/35565
https://github.com/keycloak/keycloak/issues/35566
